ddp-workspace/projects/ddp-circadia/src/app/components/header/header.component.ts

```diff
@@ -21,12 +24,14 @@ import { AppAuth0AdapterService, SDKAuth0AdapterService } from '../../services/a
 export class HeaderComponent implements OnInit, OnDestroy {
   Route = Route;
   isSticky = false;
+  isEnrollmentPaused = DDP_ENV.enrollmentPaused;
```

Comment:
that's interesting. I thought we use ConfigurationService in order to read values from DDP_ENV. What do you think?

Reply:
That's right but I don't think that we should introduce this new property in `ConfigurationService` since it's the first time we are pausing the whole study.